### PR TITLE
FY 2021 Q4 Updates

### PR DESCRIPTION
**Pull request by dwnatwick**

### CODE REVIEW

**Comment by DawnmarieDesJardins** [comment]
Please see issues #12 and #10 - part of SME review for this update.

**Comment by rdendtler** [comment]
Reviewing now, but given the new announcement, we likely need to change the overall name to Azure Virtual Desktop (https://azure.microsoft.com/en-us/blog/azure-virtual-desktop-the-desktop-and-app-virtualization-platform-for-the-hybrid-workplace/)

**Comment by DawnmarieDesJardins** [comment]
@dwnatwick 
New workshop title:  Implementing Azure Virtual Desktop in the enterprise
Thanks!

File: Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md [hunk 1]
```diff
@@ -1667,6 +1677,295 @@ For a list of example queries, refer to the following Docs article:
 
 Often during the deployment of WVD there may be challenges with either having machines join the domain or installing the agent via automation.
 
+## Exercise 9: Improving your WVD environment
+
+Duration:  60 minutes
+
+In this exercise we will our WVD expierecne bay utilizing additional features or tools with the host pools. WVD is designed to be a versatile platform to distribute and build your solution from, and these optional features allow you to enhance and/or secure the environment depending on your needs. By the end of this exercise, you will have the following features enabled:
```
Comment (rdendtler) [inline Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md:1684]: spelling error "WVD expierecne".... should be AVD experience

File: Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Hands-on lab step-by-step guide
 </div>
```
Comment (rdendtler) [inline Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md:9]: Change WVD to AVD
Change Windows Virtual Desktop to Azure Virtual Desktop

General comment: a lot of passive voice in this particular doc in Exercise 9.  Would recommend editorial review to see where this be in more active voice.  Note, not a huge problem, but makes some of the sentences awkward to me.

File: Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md [hunk 3]
```diff
@@ -1667,6 +1677,295 @@ For a list of example queries, refer to the following Docs article:
...
+|----------|:-------------:|
+| Scale session hosts using Azure Automation | https://docs.microsoft.com/en-us/azure/virtual-desktop/set-up-scaling-script |
+| Set up MSIX app attach with the Azure portal |  https://docs.microsoft.com/en-us/azure/virtual-desktop/app-attach-azure-portal |
+| Prepare an MSIX image for Windows Virtual Desktop | https://docs.microsoft.com/en-us/azure/virtual-desktop/app-attach-image-prep |
+| MSIX Packaging Tool | https://docs.microsoft.com/en-us/windows/msix/packaging-tool/tool-overview |
+| Onboard Windows 10 multi-session devices in Windows Virtual Desktop | https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/Onboard-Windows-10-multi-session-device?view=o365-worldwide |
+| Azure Cloud Shell | https://docs.microsoft.com/en-us/azure/cloud-shell/overview |
+
+
+### Task 1: Enabling autoscaling
```
Comment (rdendtler) [inline Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md:1705]: Can we call this "Enabling Autoscale"

File: Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md [hunk 4]
```diff
@@ -1667,6 +1677,295 @@ For a list of example queries, refer to the following Docs article:
...
+
+In this task, you will create an Azure Automation account and Logic App that will be regularly update the scaling of your pool based on the number of connections.  Azure Automation allows for the execution of scripts and commands within an identity related to the automation.  Logic App allows for regular execution of commands and tasks.  By combining these two resources, we allow for repeative tasks of a detailed nature which will allow us to automatically increaese and decrease the number of hosts in a WVD host pool based on demand of the environment.
```
Comment (rdendtler) [inline Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md:1707]: "...that will be regularly update.."   Should be "..that will regularly update.."

"...within an identity...".... should this be "using an identity"?

"repeative"...  should be "repititive"

File: Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md [hunk 5]
```diff
@@ -1667,6 +1677,295 @@ For a list of example queries, refer to the following Docs article:
...
+     "UseARMAPI"             = $true
+     "Location"              = $LAWorkspace.location
+     "WorkspaceName"         = $LAWorkspace.name       # Optional. If specified, Log Analytics will be used to configure the custom log table that the runbook PowerShell script can send logs to
+}
+
+.\CreateOrUpdateAzAutoAccount.ps1 @Params
+
+```
+
+1. Now that the Azure Automation account is created, keep the PowerShell window open in the background, but go to the [Azure Portal](https://portal.azure.com/) in your browser to finish its setup.
```
Comment (rdendtler) [inline Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md:1740]: "...the background, but go to the"..  Should probably be "...the background and go to the..."

Would also recommend replacing "finish its setup" with "finish the setup"

File: Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md [hunk 6]
```diff
@@ -1667,6 +1677,295 @@ For a list of example queries, refer to the following Docs article:
...
+
+1. After this script completes, go back to the [Azure Portal](https://portal.azure.com/)
+
+    ![Going back to the Azure Portal web page](images/azureportal.png "Azure Portal")
+
+1. Find the Logic App that was created by the script
+
+    ![Azure Portal Logic App](images/logicApps.png "Azure Portal")
+
+1. If you click on the **Logic app designer** link under Development Tools, you will see the graphical representation of the workflow created by the script.
```
Comment (rdendtler) [inline Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md:1826]: remove "If"..  "If you click on the... " should be "Click on the..."

File: Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md [hunk 7]
```diff
@@ -1667,6 +1677,295 @@ For a list of example queries, refer to the following Docs article:
...
+1. Click **Save** to include
+
+    ![Set the MSIX application settings and click Save](images/wvdSaveMSIXApp.png "Setup MSIX application")
+
+1. Go to the [WVD Web Client](https://rdweb.wvd.microsoft.com/arm/webclient) (or WVD client if installed locally)
+1. Click the new application icon to launch the application (refresh the page if the new application does not show up yet)
+
+This application is now running on the host pool although the application itself is not installed to the host system.  This allows for the application to also be updated by changing which MSIX package the application points to and the next time a user logs into the application. 
+
+### Task 3: Protect WVD with Defender Endpoint
```
Comment (rdendtler) [inline Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md:1930]: Product name should be Microsoft Defender for Endpoint, I beleive

File: Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md [hunk 8]
```diff
@@ -1667,6 +1677,295 @@ For a list of example queries, refer to the following Docs article:
...
+
+In this task, you will enable Defender for Endpoint service and deployment the Endpoint protection via Azure. This will allow for all systems to be protected by the Defender service from potential vulnerabilities and alert in the event of suspcious execution or activity.
```
Comment (rdendtler) [inline Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md:1932]: Please use full product name

File: Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md [hunk 9]
```diff
@@ -1667,6 +1677,295 @@ For a list of example queries, refer to the following Docs article:
...
+
+1. Selec the **Qualys** agent for deploying to Azure Defender and click **Proceed**
+
+    !["Choose the Qualys agent that is included with teh Azure Defender for servers"](images/defenderSelectQualys.png "Choose Qualys")
+
+1. Click **Fix X resources** to begin the deployment of the agent
+
+    !["Ensure the VMs expeected and click Fix X resources"](images/deployDefenderByFix.png "Fix VMs with defender")
+
+This will begin deploying Azure Defender to the Virtual Machines currently deployed.  Depending on your WVD environment, you can deploy them to systems as they are added to your domain in the WVD OU by utilizing Group Policies using the [domain group policy scenario](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/onboard-windows-10-multi-session-device?view=o365-worldwide#scenario-2-using-domain-group-policy).  Alternatively, if your hosts may be none persistent or deployed from an image, you can use the instructions for [onboarding non-persistent VDI devices](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/configure-endpoints-vdi?view=o365-worldwide).
```
Comment (rdendtler) [inline Hands-on lab/HOL step-by-step - Implementing Windows Virtual Desktop in the enterprise.md:1968]: "Alternatively, if your hosts may be none persistent or deployed from an image, you can use the instructions"

Fix awkward wording.

**Review by rdendtler — COMMENTED** [review]
In general, the new content in the HOL has some things I would like to see fixed
1. Please review for passive voice and remove where not needed
2. Please review for MCW standards.. is it "click" or is it "select".
3. Please review for spelling...found 2 spelling errors in a quick review... may be more.
4. Content/approach makes sense and the excercise is good.  Looking forward to this one!

**Comment by dwnatwick** [comment]
Pull request is ready for @DawnmarieDesJardins final QC

**Comment by dwnatwick** [comment]
@DawnmarieDesJardins.  I went through this today and we should be cleaned up.  Thank you.

**Pull request merged**